projects/fsastorefront-e2e-cypress/cypress/helpers/checkout/insurance/event-checkout.ts

```diff
@@ -139,3 +139,16 @@ export function checkEventMiniCart() {
       cy.get('.short-overview-value').should('contain.text', '€18.99');
     });
 }
+
+export function checkMiniCartWithoutOptional() {
+  const miniCartContent: sharedCheckout.MiniCart = {
+    price: ' €18.99 ',
```

Comment:
Are these spaces before and after the string left intentionally? Seems redundant.

Reply:
Valid point, removed in all places regarding mini cart